### DISCUSSION TITLE
DietPi Container LXC for Proxmox

### DISCUSSION THREAD

**Discussion by 414YogaDog** [MichaIng/DietPi#6197] [Image Request]
It would be phenomenal to have the ability to create a **Proxmox DietPi OS Container LXC** by running a command in the Proxmox Shell.

Examples of such scripts can be found at:
https://tteck.github.io/Proxmox/ 

Thank you.

**Comment by Joulinar** [comment]
Have a look to https://dietpi.com/blog/?p=2642

**Comment by MichaIng** [comment]
Closing this in favor of this request, which contains more details already: #5204
